mujoco_warp/_src/sensor.py

```diff
@@ -92,10 +132,33 @@ def _sensor_acc(m: Model, d: Data):
     objid = m.sensor_objid[accadr]
     adr = m.sensor_adr[accadr]
 
-    if sensortype == int(SensorType.ACTUATORFRC.value):
+    if sensortype == int(SensorType.ACCELEROMETER.value):
+      accelerometer = _accelerometer(m, d, worldid, objid)
+      d.sensordata[worldid, adr + 0] = accelerometer[0]
+      d.sensordata[worldid, adr + 1] = accelerometer[1]
+      d.sensordata[worldid, adr + 2] = accelerometer[2]
+    elif sensortype == int(SensorType.FORCE.value):
+      force = _force(m, d, worldid, objid)
+      d.sensordata[worldid, adr + 0] = force[0]
+      d.sensordata[worldid, adr + 1] = force[1]
+      d.sensordata[worldid, adr + 2] = force[2]
+    elif sensortype == int(SensorType.TORQUE.value):
+      torque = _torque(m, d, worldid, objid)
+      d.sensordata[worldid, adr + 0] = torque[0]
+      d.sensordata[worldid, adr + 1] = torque[1]
+      d.sensordata[worldid, adr + 2] = torque[2]
+    elif sensortype == int(SensorType.ACTUATORFRC.value):
       d.sensordata[worldid, adr] = _actuator_force(m, d, worldid, objid)
 
   if (m.sensor_acc_adr.size == 0) or (m.opt.disableflags & DisableBit.SENSOR):
     return
 
+  if wp.static(
+    np.isin(
+      m.sensor_type.numpy(),
```

Comment:
it looks like this is a cuda device array?  if so we should avoid calling `numpy` on it as it would incur a device to host transfer.  if you'd like to use it this way you should do `wp.array(..., device='cpu')` in `io.py`

However you use this field in a kernel, so it needs to be on device.  This is a scenario where it's probably OK to just have a bool param as you had before.  We will clean this up further in #148

Reply:
We discussed this offline - it's possible that actually this works just fine because `wp.static` ensures the device->host transfer is only there during graph capture but not during actual stepping, but @thowell will confirm.

we also need to make sure this is JAX friendly.